mistral/arch.cc

```diff
@@ -423,6 +423,99 @@ ArcBounds Arch::getRouteBoundingBox(WireId src, WireId dst) const
     return bounds;
 }
 
+TimingPortClass Arch::getPortTimingClass(const CellInfo *cell, IdString port, int &clockInfoCount) const
+{
+    clockInfoCount = 0;
+    if (cell->type.in(id_MISTRAL_NOT, id_MISTRAL_BUF, id_MISTRAL_ALUT2, id_MISTRAL_ALUT3, id_MISTRAL_ALUT4, id_MISTRAL_ALUT5, id_MISTRAL_ALUT6)) {
```

Comment:
worth dealing with `MISTRAL_ALUT_ARITH` too ?

Reply:
Yeah, I will. But since most of the cells are in one of these categories I figured it'd be a good start. LUTRAM timing also needs to be in there, I think.

Reply:
Yeah, makes sense